bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -1708,7 +1717,7 @@ lnd/clm2/surfdata_esmf/ctsm5.3.0/surfdata_4x5_hist_1850_16pfts_c241007.nc</fsurd
 <fsurdat hgrid="360x720cru"  sim_year="1850">
```

Comment:
I was thinking we wouldn't bring the ctsm5.4 surface datasets to master yet. Since it's labeled as a ctsm5.3 tag, you expect it to have ctsm5.3 datasets. Although the advantage of this is that we wouldn't have to do branch tags for cesm alpha tags. But, I think it's clearer if we don't and alpha tags are infrequent enough that it's OK.

So I'd revert the changes from here to the end of the file.